level-0/l0-aprims.lisp

```diff
@@ -191,9 +191,9 @@ synchronization between threads."
     (report-bad-arg rw 'read-write-lock)))
   
 
-(defun %make-semaphore-ptr ()
+(defun %make-semaphore-ptr (&key (count 0))
```

Comment:
Please change `%make-semaphore-ptr` to take a required count argument, i.e., `(defun %make-semaphore-ptr (count) ...)`, and then update the call to it in `make-semaphore` accordingly.

Reply:
Done